### ISSUE TITLE
backup firmware check?

### ISSUE THREAD

**Issue by Macrisu** [warp-more-hardware/esp32-firmware#52]
The discussion area for this project would help.
Thank you for your efforts.

I've followed the wiki and made a backup for the two chips.

The GD backup is:
esp - bin file 8.4 MB
gd - bin file - 524 KB
gd - hex file - **1.2 MB**

For GD I've used the app, not the script 

Is this correct?

AC011K-AE-25_V1.2.460.hex is just 539 KB

**Comment by Macrisu** [comment]
I've changed the ESP and GD firmware from the interface.
Those were my original software versions (ESP and GD).
![versiuni_my_originale](https://user-images.githubusercontent.com/16325429/225892290-ec77f06e-1bd0-4c90-8634-e71571266215.jpg)

**Comment by bs-github** [comment]
> The discussion area for this project would help.

Sorry, I tried to give directions on how to ask for help in the bug report templates.
As long as there is no big enough community to not only ask, but answer in the discussion area as well, I decided to keep that closed.

> I've followed the wiki and made a backup for the two chips.

👍 

> The GD backup is: esp - bin file 8.4 MB gd - bin file - 524 KB gd - hex file - **1.2 MB**
> Is this correct?

ESP sounds about right, GD as well.
Feeding GD firmware backup files (made via hardware) back via the web interface is untested (should not destroy anything - but may not work).

> AC011K-AE-25_V1.2.460.hex is just 539 KB

Explanation:  
The backup includes _empty space_ on the chip. Which means big areas of hex FF bytes. The downloadable `.hex` firmware files do skip those sections and are therefore a lot smaller.
